PUBLISHING.md

```diff
@@ -68,37 +79,87 @@ These unsigned dmgs will be used as input for the Jenkins build.
 
 Once the PR is merged and the preview build is created, follow these steps for testing and eventual release:
 
-1. Confirm the new preview version is published (do not promote as stable yet).
+### 3.1 Confirm the new preview version is published (do not promote as stable yet)
+
+### 3.2 Announce Preview Test Phase
 
-2. Initiate a discussion on GitHub using the following template:
+- Use GitHub Discussions for the announcement in the [Category General](https://github.com/eclipse-theia/theia/discussions/new?category=general).
+
+   Title:
+
+   ```md
+   Theia IDE {{1.x.z}} Preview Testing
+   ```
 
-   > Theia IDE 1.xz preview testing
-   >
-   > The new version 1.XZ.0 of the Theia IDE is available on the preview channel now. Please join the preview testing! You can download it here: {link to the download}. Update your existing installation by setting the preference *updates.channel* to *preview*. Please respond here when you can test the preview without finding blockers, by commenting with a :heavy_check_mark:. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.
+   Body:
 
-3. Announce the preview release via email to <theia-dev@eclipse.org> with the following template:
+   ```md
+   The new version {{1.x.z}} of the Theia IDE is available on the preview channel now. Please join the preview testing!
 
-   > Theia IDE 1.xz preview
-   >
-   > Hi,
-   > The new version 1.XZ.0 of the Theia IDE is available on the preview channel now. Please join the preview test and help us stabilize the release. Visit this discussion for more information and coordination: {link to the Github discussion created above}
-   >
-   > Best regards,
+   You can download it here:
+   - [Linux]()
+   - [Mac X86]()
+   - [Mac ARM]()
+   - [Windows]()
 
-4. Address reported blockers and issue patch releases (this process may take 1–2 weeks).
+   Update your existing installation by setting the preference `updates.channel` to preview.
 
-5. Once all issues are resolved, finalize the release and proceed to promote the IDE from preview to stable.
+   Please respond here when you can test the preview without finding blockers, by commenting with a ✔️.
+   If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.
+
+   | Phase | Target |  |
+   |-------|--------|----|
+   | Preview available | {{date}} | ✅ |
+   | Community preview window | {{previewStart}} → {{previewEnd}} |  |
+   | Theia IDE Promoted to Stable | {{previewEnd + 1day}} |  |
+   | Docker image Publish | {{previewEnd + 1day}} |  |
+   | Snap updated | {{previewEnd + 1day}} |  |
+   ```
+
+- Announce the preview release via email to <theia-dev@eclipse.org> with the following template:
```

Comment:
And also link this discussion on the relevant Theia release discussion (if this is based on a Theia release).